_layouts/default.html

```diff
@@ -23,7 +20,7 @@
           <div class="hero">
               {% include animated-illustration.html %}
               <div class="site-info">
-                  <a href="{{ site.baseurl }}/"><img class="site-logo" src="/images/maxwell-logo-dark-green.png"/></a>
+                  <a href="{{ site.baseurl }}/"><img alt="maxwell logo" class="site-logo" src="/images/maxwell-logo-dark-green.png"/></a>
```

Comment:
I think we want an empty alt attribute here: https://www.w3.org/WAI/tutorials/images/decorative/#decorative-image-as-part-of-a-text-link